scaffolding-python2/plan.sh

```diff
@@ -0,0 +1,38 @@
+pkg_name=scaffolding-python
```

Comment:
I might try to dry this out later with a `scaffolding-python-common` or something since we don't yet have anything which differs other than the version of python we are pinning. I'll play with it later though.